amplify/backend/function/questionslist/src/questions.js

```diff
@@ -0,0 +1,65 @@
+const AWS = require('aws-sdk');
+const uuid = require('uuid');
+const fs = require('fs');
+const parse = require('csv-parse');
+const { Config } = require('./config');
+const { Difficulty, Cost, QuestionType, textToTag } = require('./enums');
+
+AWS.config.update({ region: Config.AWS_REGION });
+
+const questionsCSV = '../data/questions.csv'
+const docClient = new AWS.DynamoDB.DocumentClient();
+
+const parseQuestions = async () => {
+  const questions = [];
+  const parser = fs
+    .createReadStream(questionsCSV)
+    .pipe(
+      parse({ delimiter: ',' })
+    );
+  for await (const row of parser) {
+    const question = {
+      title: row[4], // TODO: change this once title row in measure list
```

Comment:
Constant variables here would be great (makes it easier to change index value later on :) )